messenger.go

```diff
@@ -27,6 +25,10 @@ const (
 	MessengerProfileURL = "https://graph.facebook.com/v2.6/me/messenger_profile"
 )
 
+var (
+	defaultProfileFields = []string{"first_name", "last_name", "profile_pic", "locale", "timezone", "gender"}
```

Comment:
All these other settings variables are exported, should probably be the case here too.

Reply:
All the other settings are constants. I chose to not export this so people don't abuse it by changing it (which is not possible with const). Do you think it should be exported anyway?

Reply:
Hmm.

Not making it a constant seems logical, but I don't think users should have to dig into our source code to see what the default fields are.

I'll just edit this quickly to add a note, then merge it :)